XenAdmin/TabPages/NICPage.cs

```diff
@@ -216,13 +217,32 @@ public PIFRow(PIF pif)
                     device = PIFMetrics.device_name;
                     busPath = PIFMetrics.pci_bus_path;
                 }
-                for (int i = 0; i < 9; i++)
+                for (int i = 0; i < 10; i++)
                 {
                     Cells.Add(new DataGridViewTextBoxCell());
                     updateCell(i);
                 }
 
-            }
+                DelegatedAsyncAction action = new DelegatedAsyncAction(pif.Connection,
+                                                                        string.Format(Messages.FETCH_POSSIBLE_HOSTS, pif.Name()),

```

Comment:
As this action is not showed in the Events (suppressHistory is true), we can have empty string for title, startDescription and endDescription.